<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TpdWidgetClass</class>
 <widget class="QWidget" name="TpdWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutTPD">
     <item>
      <layout class="QHBoxLayout" name="horLayoutTpd">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>340</height>
            </size>
           </property>
           <property name="title">
            <string>TPD controls</string>
           </property>
           <widget class="QCheckBox" name="checkIsothermal">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>135</y>
              <width>101</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string>Isothermal TPD</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>14</x>
              <y>50</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Temp to:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="editDelay">
            <property name="geometry">
             <rect>
              <x>94</x>
              <y>110</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="editRate">
            <property name="geometry">
             <rect>
              <x>94</x>
              <y>80</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="editFrom">
            <property name="geometry">
             <rect>
              <x>94</x>
              <y>20</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="bnStartTpd">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>278</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Start TPD</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>14</x>
              <y>110</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Initial delay:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="bnStopTpd">
            <property name="geometry">
             <rect>
              <x>105</x>
              <y>278</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Stop TPD</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>14</x>
              <y>158</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Duration:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>164</x>
              <y>110</y>
              <width>16</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>164</x>
              <y>80</y>
              <width>21</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>K/s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>14</x>
              <y>20</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Temp from:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>164</x>
              <y>158</y>
              <width>16</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="editDuration">
            <property name="geometry">
             <rect>
              <x>94</x>
              <y>158</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>164</x>
              <y>20</y>
              <width>16</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>K</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="editTo">
            <property name="geometry">
             <rect>
              <x>94</x>
              <y>50</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>14</x>
              <y>80</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Rate:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>164</x>
              <y>50</y>
              <width>16</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>K</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="labelStatus">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>309</y>
              <width>181</width>
              <height>20</height>
             </rect>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>127</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>127</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>106</red>
                  <green>104</green>
                  <blue>100</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>TPD status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>2</x>
              <y>196</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Save folder:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="editFolder">
            <property name="geometry">
             <rect>
              <x>14</x>
              <y>216</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>247</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Experiment number:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinExpNumber">
            <property name="geometry">
             <rect>
              <x>118</x>
              <y>245</y>
              <width>51</width>
              <height>22</height>
             </rect>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
           <widget class="QPushButton" name="bnSelectFolder">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>191</y>
              <width>75</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Select...</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frameTpdChart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTempControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Temperature control</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkIsothermal</sender>
   <signal>stateChanged(int)</signal>
   <receiver>TpdWidgetClass</receiver>
   <slot>CheckIsothermalChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnStartTpd</sender>
   <signal>clicked()</signal>
   <receiver>TpdWidgetClass</receiver>
   <slot>OnStartTpdClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnStopTpd</sender>
   <signal>clicked()</signal>
   <receiver>TpdWidgetClass</receiver>
   <slot>OnStopTpdClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnSelectFolder</sender>
   <signal>clicked()</signal>
   <receiver>TpdWidgetClass</receiver>
   <slot>OnSelectFolderClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CheckIsothermalChanged(int)</slot>
  <slot>OnStartTpdClicked()</slot>
  <slot>OnStopTpdClicked()</slot>
  <slot>OnSelectFolderClicked()</slot>
 </slots>
</ui>
